<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>view::messageCenter::MessageWindow</class>
    <widget class="QWidget" name="view::messageCenter::MessageWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>394</width>
                <height>484</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MessageWindow</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QScrollArea" name="messageBoxArea">
                    <property name="widgetResizable">
                        <bool>true</bool>
                    </property>
                    <widget class="QWidget" name="messageBoxAreaItem">
                        <property name="geometry">
                            <rect>
                                <x>0</x>
                                <y>0</y>
                                <width>374</width>
                                <height>464</height>
                            </rect>
                        </property>
                    </widget>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
